src/paths.ts

```diff
@@ -25,16 +26,21 @@ export function searchUp(pathName: string, cwd: string) {
   return null;
 }
 
-export function findGitRoot(cwd: string) {
-  return searchUp('.git', cwd);
+export function findProjectRoot(cwd: string) {
+  let workspaceRoot: string | undefined;
+  try {
+    workspaceRoot = getWorkspaceRoot(cwd);
```

Comment:
I think what we need to do is:

1. leave findGitRoot() as-is because it is used as a validation check (isGitAvailable)
2. simply replace the majority of the usage of `findGitRoot(cwd)` with `getWorkspaceRoot(cwd)`

Reply:
If I replace `findGitRoot(cwd)` with `getWorkspaceRoot(cwd)`, then we loose the fallback to git root if there's no workspace in the project. Should I rather:
1. Bring back `findGitRoot()` and use it in `isGitAvailable`
2. Keep `findProjectRoot()` as it is in the PR to get the fallback functionality

Reply:
Due to time zone differences, I'll just update the branch with my suggestions and then you can approve or comment on that.

Reply:
That's fair. Let's just go with what you have.